### PR TITLE
Bug/48800 cors error on test

### PR DESCRIPTION
**Pull request by jon-shipley**
Pupil app

* bugfix for gen_config.sh
* big clean-up of pupil-spa/Dockerfile

### CODE REVIEW

File: pupil-spa/Dockerfile [hunk 1]
```diff
@@ -72,4 +54,9 @@ RUN /usr/bin/ssh-keygen -A
 
 EXPOSE 80 2222
 
+# docker-start.sh will
+#  * start sshd
+#  * generate a new config.json for the pupil-app to use using Azure env vars injected into the docker container
+#  * inject the AZURE_ACCOUNT_NAME into the fallback queue system that uses ngins to submit the completed check
```
Comment (GuyHarwood) [inline pupil-spa/Dockerfile:60]: spelling error of nginx

File: pupil-spa/docker-start.sh [hunk 2]
```diff
@@ -4,7 +4,7 @@
 /usr/sbin/sshd
 
 # Always re-generate the config based on the env variables
-sh gen_config.sh && mv config.json /usr/share/nginx/html/public
+./gen_config.sh && mv config.json /usr/share/nginx/html/assets
```
Comment (GuyHarwood) [inline pupil-spa/docker-start.sh:7]: is this exposed under the same route in the angular app?  if not, the app will need updating to reflect it

**Review by GuyHarwood — APPROVED** [review]
LGTM

just some minor points to address

File: pupil-spa/Dockerfile (hunk 1 above)
Comment (jon-shipley) [reply pupil-spa/Dockerfile]: Nice spot. Fixed.

File: pupil-spa/docker-start.sh (hunk 2 above)
Comment (jon-shipley) [reply pupil-spa/docker-start.sh:7]: The app has already been updated to use this path in the merged Angular upgrade PR.

**Review by GuyHarwood — APPROVED** [review]
LGTM

**Pull request merged**